myapp/app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+module SessionsHelper
+  def log_in(user)
+    session[:user_id] = user.id
+  end
+
+  def current_user
+    @current_user ||= User.find_by(id: session[:user_id])
+  end
+
+  def logged_in?
+    !current_user.nil?
```

Comment:
否定演算子使うぐらいなら`present`の方がいいかもしれません
```suggestion
    current_user.present?
```